### PR TITLE
Feat/qa mickey

### PR DESCRIPTION
**Pull request by m0nhawk**
### Environments
* qa-mickey

### Description of changes
* Update Atlas URL

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# qa-mickey.planx-pla.net/manifest.json
## :warning: Services on branch
- ohdsi-atlas
- ohdsi-webapi
- portal
- ws-storage

**Comment by PlanXCyborg** [comment]
# qa-mickey.planx-pla.net/manifest.json
## :warning: Services on branch
- ohdsi-atlas
- ohdsi-webapi
- portal
- ws-storage

**Review by mfshao — APPROVED** [review]

File: qa-mickey.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -30,6 +28,7 @@
     "sower": "quay.io/cdis/sower:2020.08",
     "spark": "quay.io/cdis/gen3-spark:2020.08",
     "tube": "quay.io/cdis/tube:2020.08",
+    "ws-storage": "quay.io/cdis/ws-storage:master",
```
Comment (mfshao) [inline qa-mickey.planx-pla.net/manifest.json:31]: there is `2021.01` release for wss if you want to use

Comment (m0nhawk) [reply qa-mickey.planx-pla.net/manifest.json:31]: I will update to 2021.01 later for all the services. 👍

**Pull request merged**